src/components/BaseComponents/BaseHeader.tsx

```diff
@@ -27,10 +27,12 @@ interface HeaderProps {
   title?: string;
   rightIcon?: string;
   classes: any;
+  match?: any;
```

Comment:
`match` is already included within the RouteComponentProps! You can remove this declaration instead have the interface extend RouteComponentProps:

`interface HeaderProps extends RouteComponentProps {`